### ISSUE TITLE
TG upload sometimes doesn't work right

### ISSUE THREAD

**Issue by umputun** [umputun/feed-master#78]
In some cases uploaded files displayed with 00:00 duration in telegram and doesn't work well. Some are seems to be truncated.

The solution (rather a workaround) we have discussed with @paskal  is to check if upladed file has the exceped duration prior to the message publishing and reupload (with some max attempts) if doesn't match

**Comment by umputun** [comment]
We have addressed the root cause (well, most likely) as well as several related issues:

- the sync (outside of this project) does rsync for all the files together and we may run into unfortunate timing with xml files already uploaded, but mp3 not yet, or partially available. This was addressed by splitting the sync and copying mp3 files first (to all nodes) and xmls after this 
- item's load didn't check the status code and this led to uploading the error page instead of failing on the error. This one was fixed
- to make tg upload more reliable we have added retries, up to 10 times with 1s delay
- we have optimized duration detection for items with already presented duration

All of this should resolve the issue and make tg upload great again.
